ALCF/helpers.sh

```diff
@@ -557,21 +548,27 @@ make_ds_hostfile() {
 }
 
 ezpz_savejobenv() {
-    local outfile="${WORKING_DIR}/bin/savejobenv" ; mkdir -p $(dirname "${outfile}")
-    curl -Ls https://raw.githubusercontent.com/saforem2/ezpz/main/src/ezpz/bin/savejobenv > "${outfile}" && source "${outfile}"
+    file=$(mktemp)
+    curl -Ls https://raw.githubusercontent.com/saforem2/ezpz/main/src/ezpz/bin/savejobenv > "${file}"
+    source "${file}" || exit
 }
 
 ezpz_getjobenv() {
-    local outfile="${WORKING_DIR}/bin/getjobenv" ; mkdir -p $(dirname "${outfile}")
-    curl -Ls https://raw.githubusercontent.com/saforem2/ezpz/main/src/ezpz/bin/getjobenv > "${outfile}" && source "${outfile}"
+    file=$(mktemp)
+    curl -Ls https://raw.githubusercontent.com/saforem2/ezpz/main/src/ezpz/bin/getjobenv > "${file}"
+    source "${file}" || exit
 }
 
 # +---------------------------------------+
 # | 1. Git clone ezpz (if not found)    |
 # | 2. Install ezpz (if not installed)  |
 # +---------------------------------------+
 setup_ezpz() {
-    [ -n "${PBS_NODEFILE}" ] && ezpz_savejobenv || ezpz_getjobenv
+    if [[ -n "${HOSTFILE:-${PBS_NODEFILE}}" ]]; then
+        ezpz_savejobenv
+    else
+        ezpz_getjobenv
+    fi
```

Comment:
Check and see if `${PBS_NODEFILE}` in current environment:

- if so, call `ezpz_savejobenv`
- otherwise, try calling `ezpz_getjobenv`